code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1222,7 +1222,7 @@
 	var/skipitems = get_covered_clothes()
 	var/skipbody = get_covered_body_parts()
 	. = ..()
-	if((skipbody & FACE || (skipitems & (HIDEMASK|HIDEFACE))) && (skipbody & (UPPER_TORSO | LOWER_TORSO) || (skipitems & HIDEJUMPSUIT))) //big suits/masks/helmets make it hard to tell their gender
+	if((skipbody & FACE || (skipitems & (HIDEMASK|HIDEFACE))) && ((skipbody & (UPPER_TORSO) && (skipbody & LOWER_TORSO)) || (skipitems & HIDEJUMPSUIT))) //big suits/masks/helmets make it hard to tell their gender
```

Comment:
I am not sure why the extra skipbody check is needed when something like that should be equivalent:
`&& (( skipbody & (UPPER_TORSO | LOWER_TORSO) ||`
As all you did was add an extra ( at the front